ha/core/system_health/const.py

```diff
@@ -79,54 +83,80 @@ class CONFSTORE_KEY_ATTRIBUTES(Enum):
+
 # Mapping to identify the SEVERITY based on EVENT_TYPE
 HEALTH_STATUS_TO_EVENT_SEVERITY_MAPPING = {
     "OK" : EVENT_SEVERITIES.INFORMATIONAL.value,
+    "ACTIVE" : EVENT_SEVERITIES.INFORMATIONAL.value,
+    "INACTIVE" : EVENT_SEVERITIES.WARNING.value,
     "FAULT" : EVENT_SEVERITIES.CRITICAL.value,
+    "FAILED" : EVENT_SEVERITIES.ALERT.value,
+    "DISABLED/FAILED" : EVENT_SEVERITIES.ALERT.value,
     "NONE" : EVENT_SEVERITIES.INFORMATIONAL.value,
-    "DEGRADED" : EVENT_SEVERITIES.ALERT.value
+    "NA" : EVENT_SEVERITIES.INFORMATIONAL.value,
+    "N/A" : EVENT_SEVERITIES.INFORMATIONAL.value,
+    "UNKNOWN" : EVENT_SEVERITIES.INFORMATIONAL.value,
+    "DEGRADED" : EVENT_SEVERITIES.ALERT.value,
+    "UNRECOVERABLE": EVENT_SEVERITIES.CRITICAL.value,
+    "NOT AVAILABLE": EVENT_SEVERITIES.WARNING.value
 }
 
 # Mapping resource_type received in health view schema to the one in alerts
 # Note: If the KV parsing output changes the strings in this mapping will need to be modified
 # [TBD] This mapping to be confirmed with SSPL
 RESOURCE_TYPE_MAPPING = {
-    "storage_nodes.hw.controllers" : "enclosure:hw:controller",
-    "storage_nodes.hw.disks" : "enclosure:hw:disk ",
-    "storage_nodes.hw.fanmodules" : "enclosure:hw:fan",
-    "storage_nodes.hw.psus" : "enclosure:hw:psu",
-    "storage_nodes.hw.sideplane_expander" : "enclosure:hw:sideplane",
-    "storage_nodes.fw.logical_volumes" : "enclosure:cortx:logical_volume",
-    "storage_nodes.fw.disk_groups" : "enclosure:cortx:disk_group",
-    "storage_nodes.hw.platform_sensors.temperature_sensors" : "enclosure:sensor:temperature",
-    "storage_nodes.hw.platform_sensors.voltage_sensors" : "enclosure:sensor:voltage",
-    "storage_nodes.hw.platform_sensors.current_sensors" : "enclosure:sensor:current",
-    "storage_nodes.hw.sas_ports" : "enclosure:interface:sas",
-    "compute_nodes.hw.psus" : "node:fru:psu",
-    "compute_nodes.hw.fans" : "node:fru:fan",
-    "compute_nodes.hw.disks" : "node:fru:disk",
-    "compute_nodes.hw.platform_sensors.temperature_sensors" : "node:sensor:temperature",
-    "compute_nodes.hw.platform_sensors.voltage_sensors" : "node:sensor:voltage",
-    # [TBD] this mapping should be corrected once input is received from SSPL
-    "compute_nodes.hw.platform_sensors.current_sensors" : "node",
-    "compute_nodes.hw.cpu" : "node:os:cpu",
-    "compute_nodes.hw.memory" : "node:os:memory",
-    "compute_nodes.sw.os.raid_array" : "node:os:raid_data",
-    "compute_nodes.hw.nw_ports" : "node:interface:nw",
-    "compute_nodes.hw.sas_hba" : "node:interface:sas",
-    "compute_nodes.hw.sas_ports" : "node:interface:sas:port",
-    "compute_nodes.sw.cortx_sw_services" : "node:sw:os:service",
-    "compute_nodes.sw.external_sw_services" : "node:sw:os:service",
-    "compute_nodes" : "node",
-    "storage_nodes" : "enclosure"
+    "storage.hw.controller" : "enclosure:hw:controller",
+    "storage.hw.disk" : "enclosure:hw:disk ",
+    "storage.hw.fan" : "enclosure:hw:fan",
```

Comment:
are we using it any where 
it moved to event_manager/resources.py

Reply:
is it same as we have in resources.py or different

Reply:
Its a mapping where node_health keys are getting mapped to SSPL alert keys. used in creating the heath event.

Reply:
TODO added.